server/routes/invitation.js

```diff
@@ -227,7 +225,44 @@ router.get("/user/:to_email/contacts",
   }
 );
 
+//approve the request
+router.put("/user/:to_email/approve", (req, res) => {
+   const {to_email} = req.params;
+   const {from_email} = req.body;
+   console.log('executed approval route');
+   Invitation.findOne({
+      "from_user_email": from_email,
+      "to_user_email": to_email
+   }, (err, invitation) => {
+     if(err) return console.error(err);
+     if(invitation){
+       invitation.approved=true;
+       invitation.save();
+       res.status(200).json({"approved": invitation.approved, "from_user_email": invitation.from_user_email});
+     }
+     else {
+       res.status(400).json({"error": "Could not find approved property"})
+     }
+   });
+});
```

Comment:
super similar code between `/approve` and `/reject`, we can try to modulize some of this code. the only real diff is setting `invitation.rejected` or `invitation.approved` 

it could call a helper function that reduces all the lines of repeated code and just do an if statement for approved or rejection

Reply:
Done. Thanks for suggestion Bonnie.